src/Scaffold_Command.php

```diff
@@ -366,7 +362,7 @@ function child_theme( $args, $assoc_args ) {
 		$theme_slug = $args[0];
 
 		if ( in_array( $theme_slug, array( '.', '..' ) ) ) {
-			WP_CLI::error( "Invalid theme slug specified." );
+			WP_CLI::error( "Invalid theme slug specified. The slug cannot be \".\" or \"..\"." );
```

Comment:
That's single quotes as in "Invalid theme slug specified. The slug cannot be '.' or '..'." (not the PHP string itself) I presume?!